kernel-module/Kbuild

```diff
@@ -13,9 +13,10 @@ ifdef NV_BUILD_DGPU
 	KBUILD_CFLAGS += \
 		-I$(NVIDIA_SRC_DIR) \
 		-DNV_BUILD_DGPU
+	KBUILD_EXTRA_SYMBOLS=/usr/src/nvidia-535.171.04/Module.symvers
```

Comment:
what if this is different version? can you make generic?

Reply:
This path will change as per release so user need to update it.